module/dice/ed-roll.mjs

```diff
@@ -71,6 +71,7 @@ export default class EdRoll extends Roll {
       ? Math.max( super.total, 1 )
       : super.total;
   }
+  
```

Comment:
:D

Reply:
thats me trying to place some easter eggs